pkgs/applications/misc/pdfslicer/unvendor.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/third-party/CMakeLists.txt b/third-party/CMakeLists.txt
+index 31fc934..16c2f42 100644
+--- a/third-party/CMakeLists.txt
++++ b/third-party/CMakeLists.txt
+@@ -1,13 +1,13 @@
+ add_subdirectory (adda25-threadpool)
+ add_subdirectory (catch2)
+-add_subdirectory (fmtlib)
+-add_subdirectory (GSL)
++find_package (fmt REQUIRED)
++find_package (Microsoft.GSL REQUIRED)
+ add_subdirectory (LouisCharlesC-safe)
+-add_subdirectory (spdlog)
++find_package (spdlog REQUIRED)
+ add_subdirectory (stduuid)
+ 
+ set (RANGE_V3_TESTS OFF)
+ set (RANGE_V3_EXAMPLES OFF)
+ set (RANGE_V3_PERF OFF)
+ set (RANGE_V3_HEADER_CHECKS OFF)
+-add_subdirectory (range-v3)
++find_package (range-v3 REQUIRED)
```

Comment:
According to https://stackoverflow.com/a/59035863/4446318 you would need to use add_library.

Reply:
produces the exact same output

Reply:
I poked at this as well, and couldn't get it to work (although I don't know anything about CMake). I suspect someone whose proficient with CMake could whip this into shape.